client/elife-theme/src/elements/ui/Button.js

```diff
@@ -60,8 +60,7 @@ const attention = css`
 export default css`
   font-size: ${th('fontSizeBaseSmall')}
   line-height: ${th('lineHeightBaseSmall')};
-  padding: ${th('space.2')};
-  padding-bottom: 9px;
+  padding: 12px 18px 9px 18px;
```

Comment:
Should we express these in terms of grid units? @Cooryd ?

Reply:
I think it's ok in that way @diversemix